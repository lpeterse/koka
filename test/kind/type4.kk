type list<a> {
  con Nil
  con Cons( head : a, tail : list<a> )
}
