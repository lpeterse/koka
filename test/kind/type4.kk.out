kind/type4/list: V -> V